contents/customers/mintlify.md

```diff
@@ -0,0 +1,58 @@
+---
+title: How Mintlify launched user-facing analytics, powered by PostHog
+customer: Mintlify
+logo: ../images/customers/mintlify/logo.png
+featuredImage: ../images/customers/mintlify/featured.png
+industries:
+    - SaaS, DevTool
+users:
+    - Leadership
+    - Engineering
+    - Product
+    - Users
+toolsUsed:
+    - Session Recording
+    - Autocapture
+    - PostHog Cloud
+    - API
+---
+
+Mintlify is a docs startup used by teams at [Y Combinator](/customers/ycombinator), Explo and  MindsDB to power their product and technical documentation. It helps teams with rapidly evolving products to create rich, well structured docs which convert users and empower colleagues. 
+
+Mintlify has used PostHog internally since launch, making regular use of both the product analytics and session recording tools to help it build better products. 
```

Comment:
```suggestion
Mintlify has used PostHog internally since launch, making regular use of both the product analytics and session replay tools to help it build better products. 
```